activesupport/lib/active_support/broadcast_logger.rb

```diff
@@ -218,6 +218,14 @@ def fatal!
       dispatch { |logger| logger.fatal! }
     end
 
+    def initialize_copy(other)
+      @broadcasts = []
+      @progname = other.progname.dup
+      @formatter = other.formatter.dup
+
+      broadcast_to(*other.broadcasts.map(&:dup))
```

Comment:
Late nitpick, but I don't get why this was better than doing `@broadcasts = other.broadcasts.map(&:dup)` in the first line. 🤔